pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -970,3 +972,36 @@ message StatusResponse {
     // Status of the Runtime.
     RuntimeStatus status = 1;
 }
+
+message ImageFsInfoRequest {}
+
+// UInt64Value is the wrapper of uint64.
+message UInt64Value {
+    // The value.
+    uint64 value = 1;
+}
+
+// FsInfo is the filesystem information of the images.
+message FsInfo {
+    // The block device name associated with the filesystem.
+    string device = 1;
+    // Path where the filesystem is mounted.
+    string mountpoint = 2;
```

Comment:
Maybe just the `path` is enough? Do we really need to know the mount point, or just root directory for the images? I think latter is sufficient. 

/cc @vishh